.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 
 cache: pip
 
-dist: trusty
+dist: xenial
```

Comment:
AFAIK travis does not have a `xenial` image. According to their [docs](https://docs.travis-ci.com/user/reference/overview/) only precise and trusty are available. Also the [test's logs](https://travis-ci.org/geopython/pycsw/jobs/306947343#L9) show that in fact trusty is being used and not xenial. So this should be reverted back to trusty` it seems.

Reply:
Thanks @ricardogsilva; reverted.

Reply:
Update: Travis now has a xenial image, reverted back.  This helps with Python 3.7 testing.